sdk/src/wallet/operations/transaction/send_outputs.rs

```diff
@@ -71,6 +71,7 @@ where
     ) -> Result<PreparedTransactionData, WalletError> {
         log::debug!("[TRANSACTION] prepare_send_outputs");
         let options = options.into().unwrap_or_default();
+        log::debug!("prepare_send_outputs {options:#?}");
```

Comment:
Just checking you want this to stay

Reply:
Yes, I think it could help us in the future